### PR TITLE
PMM-5317 Skip unavailable metrics scrape on macOS

### PR DESCRIPTION
**Pull request by artemgavrilov**

### CODE REVIEW

File: services/prometheus/scrape_configs.go [hunk 1]
```diff
@@ -241,43 +241,52 @@ func scrapeConfigForRDSExporter(intervalName string, interval time.Duration, hos
...
+			"standard.go",
+			"standard.process",
+			"stat",
+			"textfile.hr",
+			"vmstat",
+		)
+	}
+
+	hr, err = scrapeConfigForStandardExporter("hr", s.HR, params, append(hrCollect,
 		"cpu",
```
Comment (BupycHuk) [inline services/prometheus/scrape_configs.go]: Do only 7 collectors work on MacOS?

Comment (artemgavrilov) [reply services/prometheus/scrape_configs.go]: Yep, only 7 from specified in that file. `textfile.*` can work on macOS by itself, but it's disabled.
https://github.com/percona/pmm-managed/blob/PMM-2.0/services/agents/node.go#L52

Comment (AlekSi) [reply services/prometheus/scrape_configs.go]: But where we enable CPU collector on macOS, for example?

Comment (AlekSi) [reply services/prometheus/scrape_configs.go]: Ah, it is enabled by default.

**Pull request merged**